<?xml version="1.0"?>
<launch>
    <node pkg="twist_controller" type="dbw_node.py" name="dbw_node" output="screen">
        <param name="vehicle_mass" value="1080." />
        <param name="fuel_capacity" value="0." />
        <param name="brake_deadband" value=".2" />
        <param name="decel_limit" value="-5." />
        <param name="accel_limit" value="1." />
        <param name="wheel_radius" value="0.335" />
        <param name="wheel_base" value="3" />
        <param name="steer_ratio" value="14.8" />
        <param name="max_lat_accel" value="3." />
        <param name="max_steer_angle" value="8." />
    </node>
</launch>
